open-company/open-company-web
# Search

Adds a search box in the top left and uses the search service to find results of posts.

To test:

- make sure search service is running locally
- make sure some data is indexed by editing or creating a post.
- input more than one character in the search box
- [x] does the empty result spy glass image display?
- [x] does clicking outside of the search box remove the spy glass?
- [x] does clicking inside the search box redisplay the spy glass?
- input a search term that shows results
- [x] does the expected result get displayed?
- click on result
- [x] does the post open?

- [x] before deploying to beta make sure we have a successfully deployed search service to beta
- [ ] deploy to beta
